src/index.md

```diff
@@ -1,59 +1,26 @@
 # Welcome to the DataJoint Documentation
 
-<h2> DataJoint Ecosystem </h2>
+## DataJoint Core 
```

Comment:
What motivated the choice to remove cards? The [elements doc page](https://datajoint.com/docs/elements/) takes this approach. I'm wondering if we should take the same approach there

Reply:
TBD in subsequent PR's.